### DISCUSSION TITLE
Warning: Container must join at least one network, running on Synology 920+ DSM 7.0.1

### DISCUSSION THREAD

**Discussion by professordave** [qdm12/gluetun#947] [Q&A] [answered]
Hello,

I have the Gluetun container configured and working. 

I have a qbittorent container that uses Gluetun for all network traffic.

I can start and stop the qbittorent container from the command line using the docker commands.

However, if I try and start the container using the Synology Docker UI, it complains
"Container must join at least one networrk"

It would be great if I can also start this container from the synology docker UI.

Is there some options I am missing? Is this related to running docker on the synology?

**Comment by bozzfozz** [comment, marked as answer]
you have to use portainer as docker ui , synology's one is not really helpful

**Reply by bozzfozz** [reply]
and before I switched to portainer , I had to go through
 synology task manager and run it as a script. That's the only way it worked

**Reply by qdm12** [reply]
Thanks @bozzfozz for helping out answering! :medal_military:

**Comment by professordave** [comment]
Thanks for the prompt reply. So I stopped the Qbittorrent container, but then it is no longer in the list as a stopped container. It disappears from the list.

it is still visible in the synology docker ui, but it refuses to start it.

I can start it from the docker command line.

**Reply by professordave** [reply]
Portainer works for me now! Thanks
